kotlin/internal/jvm/compile.bzl

```diff
@@ -398,8 +389,11 @@ def _run_kt_builder_action(
     args.add_all("--sources", srcs.all_srcs, omit_if_empty = True)
     args.add_all("--source_jars", srcs.src_jars + generated_src_jars, omit_if_empty = True)
     args.add_all("--deps_artifacts", deps_artifacts, omit_if_empty = True)
-
-    args.add_joined("--kotlin_friend_paths", friend.paths, join_with = "\n")
+    args.add_joined(
+        "--kotlin_friend_paths",
+        _associate_utils.get_jars(associates),
```

Comment:
Use add_all(map_each), not depset.to_list. This can eat up a lot of memory during analysis phase.

Reply:
Done